app/src/features/surveys/components/MethodForm.tsx

```diff
@@ -0,0 +1,207 @@
+            onChange={handleChange}
+            sx={{ width: '100%', backgroundColor: '#fff' }}>
+            {codesContext.codesDataLoader.data.sample_methods.map((item) => (
+              <MenuItem key={item.id} value={item.id}>
+                {item.name}
+              </MenuItem>
+            ))}
+          </Select>
+          <FormHelperText>{get(touched, 'method_lookup_id') && get(errors, 'method_lookup_id')}</FormHelperText>
+        </FormControl>
+      </Box>
+      <Box mb={3}>
+        <CustomTextField
+          name="description"
+          label="Description"
+          maxLength={250}
+          other={{ multiline: true, placeholder: 'Maximum 250 characters', rows: 3 }}
+        />
+      </Box>
+
+      <Box component="fieldset">
+        <Typography
+          component="legend"
+          id="time_periods"
+          sx={{
+            mb: 1
+          }}>
+          Time Periods
+        </Typography>
+        <Box>
+          <FieldArray
+            name="periods"
+            render={(arrayHelpers: FieldArrayRenderProps) => (
+              <>
+                <List disablePadding>
+                  {values.periods?.map((item, index) => {
+                    return (
+                      <ListItem
+                        alignItems="flex-start"
+                        disableGutters
+                        key={`${item.start_date}-${item.end_date}`}
+                        sx={{
+                          '& .MuiListItemSecondaryAction-root': {
+                            top: '36px'
+                          }
+                        }}>
+                        <Box width="100%">
+                          <StartEndDateFields
+                            formikProps={formikProps}
+                            startName={`periods[${index}].start_date`}
+                            endName={`periods[${index}].end_date`}
+                            startRequired={true}
+                            endRequired={true}
+                          />
+                        </Box>
+                        <ListItemSecondaryAction
+                          sx={{
+                            top: '36px'
+                          }}>
+                          <IconButton
+                            data-testid="delete-icon"
+                            aria-label="remove time period"
+                            onClick={() => arrayHelpers.remove(index)}>
+                            <Icon path={mdiTrashCanOutline} size={1} />
+                          </IconButton>
+                        </ListItemSecondaryAction>
+                      </ListItem>
+                    );
+                  })}
+                </List>
+                {errors.periods && !Array.isArray(errors.periods) && (
+                  <Box pt={2}>
+                    <Typography style={{ fontSize: '12px', color: '#f44336' }}>{errors.periods}</Typography>
```

Comment:
Rather than using `style`, can you set the size by using a `variant` e.g. `vairant="body2"`? And can you pass the color directly as a prop, e.g. `color="#F44336"`?

Reply:
this pattern from another error display in the app

Reply:
good call